bindings/zeebe/command/activate_jobs.go

```diff
@@ -0,0 +1,78 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package command
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/dapr/components-contrib/metadata"
+)
+
+const (
+	// errors
+	missingJobTypeErrorMsg           = "jobType is a required attribute"
```

Comment:
It's better to have these as error variables. Example:

```golang
var (
    ErrMissingJobType           = errors.New("jobType is a required attribute")
    ErrMissingMaxJobsToActivate = errors.New("maxJobsToActivate is a required attribute")
)
```

Reply:
Fixed